pycaching/cache.py

```diff
@@ -831,10 +918,14 @@ def __init__(self, id=None, type=None, location=None, note=None):
         self._note = note
 
     @classmethod
-    def from_html(cls, root):
-        """Return a dictionary of all waypoints found in the page representation"""
+    def from_html(cls, root, id_="ctl00_ContentBody_Waypoints"):
```

Comment:
Please remove the default value of `id_` and propagate it to the method call (line 650). Also maybe rename the variables `root` and `id` somehow more meaningful...